src/libcore/str/mod.rs

```diff
@@ -299,6 +299,20 @@ unsafe fn from_raw_parts_mut<'a>(p: *mut u8, len: usize) -> &'a mut str {
 ///
 /// assert_eq!("💖", sparkle_heart);
 /// ```
+///
+/// Incorrect bytes:
+///
+/// ```
```

Comment:
`unchecked ` means it won't return an `Err`. It won't panic either, but the call to `println!` will, as it attempts to print an invalid string. I'll split the lines out to make that clearer.